### ISSUE TITLE
Custom Themes: Enable Changing Just Font Color

### ISSUE THREAD

**Issue by jonasms** [warpdotdev/warp#2019]
### Describe the solution you'd like?

```text
Changing `foreground` changes the font color as well as the background color.

For example, setting `foreground` to a green adds green to all of the elements (e.g. blocks) as well.

Afaict, it's not possible to have a dark black background with a green font color.
```

### How important is this feature to you?

3

**Comment by dannyneira** [comment]
Thanks for this feature request! Please +1 this request as it helps us gauge interest. We will post any updates here.

**Comment by will-rodriguez** [comment]
Looks like this is a duplicate of #409

**Comment by reikrom** [comment]
+1 
I'd like to be able to change the background.
Atm there are 2 options. Blinding light mode, or pure black dark mode where it's hard to see where the CLI starts and ends depending on the viewport background.

**Comment by jimcornmell** [comment]
Yep, I can change the font colour to what I want, i.e. green.  But it also changes popups/panels etc.  Users must be able to decide what font colour they need, and panels.  They should not be the same setting!  In this example the text colour is fine, but the windows look awful.

![Screenshot_20240226_101841](https://github.com/warpdotdev/Warp/assets/7923416/43176e74-531f-48d5-a247-b6f38d2b279d)

**Comment by alxvallejo** [comment]
It's insane this feature request is still open
